tests/acceptance/features/cliDataExporter/import.feature

```diff
@@ -21,3 +21,32 @@ Feature: An administrator wants to import a user using the commandline
     And as "usertrash" file "AFolder/DeletedFile.txt" should exist
     And the content of file "AFolder/DeletedFolder/fileinfolder.txt" for user "usertrash" should be "text in file in deleted folder"
     And the content of file "AFolder/DeletedFile.txt" for user "usertrash" should be "text in deleted file"
+
+  Scenario: Restore and rename files in trashbin after import
+    Given a user has been imported from path "trashbinExport/usertrash" using the occ command
```

Comment:
I am kinda confused here. Why is usertrash available for import in the beginning of the scenario here? There is not any export in the Background? am I missing something?

Reply:
The exported data for `usertrash` is already available inside `tests/acceptance/data/trashbinExport` that is why export for this user has not been mentioned in the background.